### ISSUE TITLE
The minitbatch size is fixed to 32. I cant change!!

### ISSUE THREAD

**Issue by CWD4RKC0D3R** [Microsoft/CNTK#3364]
Hi, I load data from numpy array and train network according to that data (not reading from file). the problem is that even i tried changing the size of minibatch. it wasnt changed. 

the code that you need to see:

`
        xtrain = scipy.sparse.csr_matrix((xtrain), shape=(N, input_dim_lr))
        ytrain = scipy.sparse.csr_matrix((ytrain), shape=(N, num_classes_lr))
        learning_rate = 0.001
        learning_rate = [learning_rate]
        lr_schedule = C.learning_rate_schedule(learning_rate, C.UnitType.minibatch)

    loss = C.squared_error(z, l)
    error = C.classification_error(z, l)

    learner = C.adam(z.parameters, lr = lr_schedule,
                          momentum = C.momentum_as_time_constant_schedule(700), minibatch_size=1000)

    progress_writer = C.logging.ProgressPrinter(1000000)

    train_summary = loss.train((xtrain, ytrain), parameter_learners=[learner],
                       callbacks=[progress_writer])`

**Comment by CWD4RKC0D3R** [comment]
I found the solution
